tagger/preset.py

```diff
@@ -106,3 +108,8 @@ def list(self) -> List[str]:
             presets.append(self.default_filename)
 
         return presets
+
+
+preset = Preset(Path(
+    os.path.join(extensions_dir, 'stable-diffusion-webui-wd14-tagger/presets')
```

Comment:
refer to [AUTOMATIC1111/stable-diffusion-webui/wiki/Developing-extensions](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Developing-extensions)
```python
from modules import scripts
extensions_dir = str(scripts.basedir())
```
And, `"/"` is for linux, but `"\"` is for windows

Reply:
I tried this in first instance, but somehow got to a folder, 'stable-diffusion-webui/presets'

Reply:
(same person BTW) I'm guessing you call this from your app, and the scripts basedir might be the path where you invoke your program or it's because the tagger does not live in a scripts dir.

Reply:
yes that will work also, I will adapt it. Is extension_dir problematic?

Reply:
`extension_dir` is ok. But `"stable-diffusion-webui-wd14-tagger"` may be  problematic.

Because users may change the name of the folder

Reply:
I have it adapted, amongst other changes, will add the changes later today and merge, if all is well.